easybuild/easyconfigs/h/heFFTe/heFFTe-2.3.0-cpeGNU-22.12-ROCm.eb

```
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'HeFFTe'
+version = '2.3.0'
+versionsuffix = '-ROCm'
```

```suggestion
versionsuffix = '-rocm-5.2'
```
Same comment as for the cpeAMD version